server/java/src/main/java/com/stripe/sample/Server.java

```diff
@@ -62,9 +61,9 @@ public static void main(String[] args) {
             String accountID = account.getId();
 
             AccountLinkCreateParams createAccountLinkParams = new AccountLinkCreateParams.Builder()
-                    .setAccount(accountID).setType("onboarding")
-                    .setFailureUrl(String.format("%s/onboard-user/refresh", origin))
-                    .setSuccessUrl(String.format("%s/success.html", origin)).build();
+                    .setAccount(accountID).setType("account_onboarding")
```

Comment:
Library does use enums for string values?

Reply:
It does, but it takes a String. I just work here, though.

https://github.com/stripe/stripe-java/blob/19603398bf8f29431f3c5630a2b9c6bb19c157b0/src/main/java/com/stripe/param/AccountLinkCreateParams.java#L258

Reply:
Right. For consistency we should probably use the ENUM in the builder: https://stripe.dev/stripe-java/com/stripe/param/AccountLinkCreateParams.Type.html

`AccountLinkCreateParams.Type.ACCOUNT_ONBOARDING`